nextcloud-startup-script.sh

```diff
@@ -625,12 +545,6 @@ mesg n
 
 ROOTNEWPROFILE
 
-# Download all app scripts
-print_text_in_color "$ICyan" "Downloading all the latest app scripts to $SCRIPTS/apps..."
-mkdir -p $SCRIPTS/apps
-cd $SCRIPTS/apps
-check_command curl -s https://codeload.github.com/nextcloud/vm/tar.gz/master | tar -xz --strip=2 vm-master/apps
```

Comment:
If we don't delete these lines, the apps.sh would take the apps, that are already installed and we then shouldn't have to make the scripts backwards compatible.
This could only be a problem if we add a new script...

Reply:
But I would really like this in the current form, since it always would load the newest scripts and you have a main menu for installing apps.